### PR TITLE
Create emploee component page + test for add emploee

### PR DESCRIPTION
**Pull request by SerhiiTarasiuk**

### CODE REVIEW

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmploeeComponent.cs [hunk 1]
```diff
@@ -1,35 +1,30 @@
...
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestPages;
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
 
-namespace EasyRestProjectNetTeam2.EasyRestPages
+namespace EasyRestProjectNetTeam2.EasyRestComponentsObj
 {
-    public class AdministratorsManagePage : BasePage
+    public class AddEmploeeComponent : BasePage
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmploeeComponent.cs]: please rename to  AddEmplo**y**(!!!)eeComponent

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+                dataModel.EmailForNewEmploee, dataModel.PasswordForNewEmploee,
+                dataModel.PhoneForNewEmploee, dataModel.TimeToWait);
+            manageAdministratorPage.AddEmploeeComponent.ClickAddNewEmploee();
+            var actualPageUrl = manageAdministratorPage.WaitAndGetTextFromAdministratorNameField(dataModel.TimeToWait);
+            var expectedSearchWord = dataModel.NameForNewEmploee;
+            StringAssert.Contains(expectedSearchWord, actualPageUrl, "Search word is absent in emploee name field");
+        }
+
+        [Test]
+        public void CheckAddWaiter()
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: you have duplicate logic here, please think how you can avoid the duplication

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+            var expectedSearchWord = dataModel.NameForNewEmploee;
+            StringAssert.Contains(expectedSearchWord, actualPageUrl, "Search word is absent in emploee name field");
+        }
+
+        [TearDown]
+        public override void TearDown()
+        {
+            DatabaseManager.SendNonQuery(queryDataModel.SetPreviousAdministrator);
+            DatabaseManager.SendNonQuery(queryDataModel.DeleteUserByEmail, dataModel.EmailForNewEmploee);
+            DatabaseManager.SendNonQuery(queryDataModel.DeleteTokenByEmail, dataModel.EmailForOwner);
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: are these methods exception safe? is it possible that `SendNonQuery()` might fail with exception and `driver.Close();` won't work?

Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: Yes, they are safe. There is one condition that DB must be reset and everybody in our team has done this so all condition is true.

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmploeeComponent.cs (hunk 1 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmploeeComponent.cs]: Done

**Review by AhnessP — COMMENTED** [review]
typo in class name Employee

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs (hunk 2 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: The second test was not stable. I change assert.

File: EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+
+        public ManageAdministratorPage(IWebDriver driver) : base(driver) { }
+
+        [FindsBy(How = How.XPath, Using = "//button[@title='Add Administrator']")]
+        private IWebElement _plusAdministratorButton;
+
+        [FindsBy(How = How.XPath, Using = "(//button[contains(@class, 'MuiButtonBase-root')])[2]")]
+        private IWebElement _deleteAdministratorButton;
+
+        [FindsBy(How = How.XPath, Using = "(//span[contains(@class, 'MuiTypography-root-41 MuiTypography-body1')])[5]")]
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs]: I'm not sure that this locator would work. You have number 41, which is dynamic, I suppose.

File: EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs [hunk 5]
```diff
@@ -0,0 +1,40 @@
+﻿using EasyRestProjectNetTeam2.Decorator;
+using EasyRestProjectNetTeam2.EasyRestComponentsObj;
+using OpenQA.Selenium;
+using SeleniumExtras.PageObjects;
+
+namespace EasyRestProjectNetTeam2.EasyRestPages
+{
+    public class ManageAdministratorPage : BasePage
+    {
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs]: please rename to AddEmplo**y**eeComponent

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 6]
```diff
@@ -1,10 +1,16 @@
...
+using OpenQA.Selenium;
 using SeleniumExtras.PageObjects;
+using System.Collections.Generic;
 
 namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class ManageWaitersPage : BasePage
     {
+
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: please rename to AddEmplo**y**eeComponent

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 7]
```diff
@@ -16,32 +22,22 @@ public ManageWaitersPage(IWebDriver driver) : base(driver)
...
-        private IWebElement _inputEmailValidationWarning;
+        [FindsBy(How = How.XPath, Using = "//div[contains(@class, 'MuiPaper-rounded')]/ul/li/div/span")]
+        private IList<IWebElement> _listOfWaiters;
 
-        [FindsBy(How = How.XPath, Using = "//p[text()='Password is required']")]
-        private IWebElement _inputPasswordValidationWarning;
-
-        [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
-        private IWebElement _inputPhoneNumberValidationWarning;
+        public bool CheckThatNewWaiterAppears(string NameForNewEmploee)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: you can simplify this method:

```
public bool CheckThatNewWaiterAppears(string NameForNewEmploee)
{
            driver.Navigate().Refresh();
            return _listOfWaiters.Any(waiterElement => waiterElement.Text.Equals(NameForNewEmploee);
}
```

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 8]
```diff
@@ -16,32 +22,22 @@ public ManageWaitersPage(IWebDriver driver) : base(driver)
...
+        {
+            driver.Navigate().Refresh();
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: After refreshing the page, usually, you should wait some time until the page is reloaded. Does it work stable?

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 9]
```diff
@@ -0,0 +1,71 @@
...
+            menuPage.LeftBarComponent.WaitAndClickAdministratorsLeftBarButton(dataModel.TimeToWait);
+            manageAdministratorPage = GetManageAdministratorPage();
+            manageAdministratorPage.ClickPlusAdministrator();
+            manageAdministratorPage.AddEmploeeComponent.WaitAndSendKeysToInputData(dataModel.NameForNewEmploee,
+                dataModel.EmailForNewEmploee, dataModel.PasswordForNewEmploee,
+                dataModel.PhoneForNewEmploee, dataModel.TimeToWait);
+            manageAdministratorPage.AddEmploeeComponent.ClickAddNewEmploee();
+            var actualPageUrl = manageAdministratorPage.WaitAndGetTextFromAdministratorNameField(dataModel.TimeToWait);
+            var expectedSearchWord = dataModel.NameForNewEmploee;
+            StringAssert.Contains(expectedSearchWord, actualPageUrl, "Search word is absent in emploee name field");
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: emplo**y**ee

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 10]
```diff
@@ -0,0 +1,71 @@
...
+
+        [Test]
+        [Category("(ow) Possibility to manage waiters")]
+        public void CheckAddAdministratorWithValidData()
+        {
+            menuPage.LeftBarComponent.WaitAndClickAdministratorsLeftBarButton(dataModel.TimeToWait);
+            manageAdministratorPage = GetManageAdministratorPage();
+            manageAdministratorPage.ClickPlusAdministrator();
+            manageAdministratorPage.AddEmploeeComponent.WaitAndSendKeysToInputData(dataModel.NameForNewEmploee,
+                dataModel.EmailForNewEmploee, dataModel.PasswordForNewEmploee,
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: PasswordForNewEmplo**y**ee

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 11]
```diff
@@ -0,0 +1,71 @@
...
+        }
+
+        [Test]
+        [Category("(ow) Possibility to manage waiters")]
+        public void CheckAddAdministratorWithValidData()
+        {
+            menuPage.LeftBarComponent.WaitAndClickAdministratorsLeftBarButton(dataModel.TimeToWait);
+            manageAdministratorPage = GetManageAdministratorPage();
+            manageAdministratorPage.ClickPlusAdministrator();
+            manageAdministratorPage.AddEmploeeComponent.WaitAndSendKeysToInputData(dataModel.NameForNewEmploee,
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: NameForNewEmplo**y**ee

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 12]
```diff
@@ -0,0 +1,71 @@
...
+                dataModel.EmailForNewEmploee, dataModel.PasswordForNewEmploee,
+                dataModel.PhoneForNewEmploee, dataModel.TimeToWait);
+            manageAdministratorPage.AddEmploeeComponent.ClickAddNewEmploee();
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: ClickAddNewEmplo**y**ee

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs [hunk 13]
```diff
@@ -0,0 +1,71 @@
...
+        {
+            menuPage.LeftBarComponent.WaitAndClickWaiterLeftBarButton(dataModel.TimeToWait);
+            manageWaitersPage = GetManageWaitersPage();
+            manageWaitersPage.ClickAddWaiterButton();
+            manageWaitersPage.AddEmploeeComponent.WaitAndSendKeysToInputData(dataModel.NameForNewEmploee,
+                dataModel.EmailForNewEmploee, dataModel.PasswordForNewEmploee,
+                dataModel.PhoneForNewEmploee, dataModel.TimeToWait);
+            manageWaitersPage.AddEmploeeComponent.ClickAddNewEmploee();
+            var ifNewWaiterAppears = manageWaitersPage.CheckThatNewWaiterAppears(dataModel.NameForNewEmploee);
+            Assert.IsTrue((ifNewWaiterAppears), "No Waiter in the list");
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: Please edit your error message; it should explain to everyone the meaning of your requirement (or what you are trying to assert) and not explain what you are trying to implement, e.g., "New Waiter is not added."

File: EasyRestProjectNetTeam2/Models/DataModel.cs [hunk 14]
```diff
@@ -15,5 +15,11 @@ public class DataModel
         public string NavigationHotCatagoryMenuPage { get; set; }
         public string NavigationSoupCatagoryMenuPage { get; set; }
         public string NavigationCoctailsCatagoryMenuPage { get; set; }
+        public string EmailForOwner { get; set; }
+        public string PasswordForOwner { get; set; }
+        public string NameForNewEmploee { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/Models/DataModel.cs]: NameForNewEmplo**y**ee

File: EasyRestProjectNetTeam2/Models/DataModel.cs [hunk 15]
```diff
@@ -15,5 +15,11 @@ public class DataModel
...
+        public string EmailForNewEmploee { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/Models/DataModel.cs]: Emplo**y**ee

File: EasyRestProjectNetTeam2/Models/DataModel.cs [hunk 16]
```diff
@@ -15,5 +15,11 @@ public class DataModel
...
+        public string PasswordForNewEmploee { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/Models/DataModel.cs]: Emplo**y**ee

File: EasyRestProjectNetTeam2/Models/DataModel.cs [hunk 17]
```diff
@@ -15,5 +15,11 @@ public class DataModel
...
+        public string PhoneForNewEmploee { get; set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/Models/DataModel.cs]: Emplo**y**ee

**Review by dmatsev — CHANGES_REQUESTED** [review]
Please resolve the comments.

File: EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs (hunk 5 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs]: Done.

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs (hunk 6 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: Done.

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs (hunk 9 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: Done.

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs (hunk 10 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: Done.

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs (hunk 12 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: Done.

File: EasyRestProjectNetTeam2/Models/DataModel.cs (hunk 14 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/Models/DataModel.cs]: Done.

File: EasyRestProjectNetTeam2/Models/DataModel.cs (hunk 15 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/Models/DataModel.cs]: Done.

File: EasyRestProjectNetTeam2/Models/DataModel.cs (hunk 17 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/Models/DataModel.cs]: Done.

File: EasyRestProjectNetTeam2/Models/DataModel.cs (hunk 16 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/Models/DataModel.cs]: Done.

File: EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs (hunk 4 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestPages/ManageAdministratorPage.cs]: This locator works 'cause dynamic part is deleted[](url)
d1

But I made this shorter.
Screenshot_20

File: EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs (hunk 13 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestTests/CheckAddEmploeeTest.cs]: done

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs (hunk 7 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: thnx

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs (hunk 8 above)
Comment (SerhiiTarasiuk) [reply EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: Yes, U are right. I've tried to do tests when some have app load my PC, success result was 8 from 10 times test fall down. so I try to add waiter but it was wrong decision then I added one more assert if pop up with confirmation displayed and it gives enough time for checking.

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs [hunk 18]
```diff
@@ -43,40 +38,52 @@ public AdministratorsManagePage(IWebDriver driver) : base(driver)
         [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
         private IWebElement _inputPhoneNumberValidationWarning;
 
+        public void WaitAndSendKeysToInputName(string name, int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs]: please rename the parameter to **t**imeToWait, parameter's name begins with lower case

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs [hunk 19]
```diff
@@ -43,40 +38,52 @@ public AdministratorsManagePage(IWebDriver driver) : base(driver)
...
+        {
+            _inputName.WaitAndSendKeys(driver, TimeToWait, name);
+        }
+
+        public void WaitAndSendKeysToInputEmail(string email, int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs]: please rename the parameter to **t**imeToWait, parameter's name begins with lower case

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs [hunk 20]
```diff
@@ -43,40 +38,52 @@ public AdministratorsManagePage(IWebDriver driver) : base(driver)
...
+        {
+            _inputEmail.WaitAndSendKeys(driver, TimeToWait, email);
+        }
 
-        public void ClickDeleteAdministrator()
+        public void WaitAndSendKeysToInputPassword(string password, int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs]: please rename the parameter to **t**imeToWait, parameter's name begins with lower case

File: EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs [hunk 21]
```diff
@@ -43,40 +38,52 @@ public AdministratorsManagePage(IWebDriver driver) : base(driver)
...
         {
-            _deleteAdministratorButton.Click();
+            _inputPassword.WaitAndSendKeys(driver, TimeToWait, password);
         }
 
-        public void SendKeysToInputName(string name)
+        public void WaitAndSendKeysToInputPhoneNumber(string phonenumber, int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestComponentsObj/AddEmployeeComponent.cs]: please rename the parameter to **t**imeToWait, parameter's name begins with lower case

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 22]
```diff
@@ -1,47 +1,37 @@
...
 using SeleniumExtras.PageObjects;
+using System.Collections.Generic;
+using System.Linq;
 
 namespace EasyRestProjectNetTeam2.EasyRestPages
 {
     public class ManageWaitersPage : BasePage
     {
+
+        public AddEmployeeComponent AddEmploeeComponent { get; private set; }
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: AddEmplo**y**eeComponent

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 23]
```diff
@@ -1,47 +1,37 @@
...
-        private IWebElement _inputEmailValidationWarning;
-
-        [FindsBy(How = How.XPath, Using = "//p[text()='Password is required']")]
-        private IWebElement _inputPasswordValidationWarning;
+        [FindsBy(How = How.XPath, Using = "//button[@title='Add Waiter']")]
+        private IWebElement _addWaiterButton;
 
-        [FindsBy(How = How.XPath, Using = "//p[text()='Phone number is required']")]
-        private IWebElement _inputPhoneNumberValidationWarning;
+        public bool CheckThatNewWaiterAppears(string NameForNewEmployee)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: **n**ameForNewEmployee

File: EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs [hunk 24]
```diff
@@ -51,44 +41,13 @@ public void ClickDeleteButton()
...
-        {
-            return _inputEmailValidationWarning.Displayed;
-        }
-        public bool IsInputPasswordValidationWarningExist()
-        {
-            return _inputPasswordValidationWarning.Displayed;
-        }
-        public bool IsInputPhoneNumberValidationWarningExist()
+
+        public bool WaitAndCheckIfDisplayedUserSuccesfullyAddedConfirmationPopUp(int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/ManageWaitersPage.cs]: **t**imeToWait

File: EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs [hunk 25]
```diff
@@ -34,21 +35,21 @@ public void WaitAndClickJonsonDetails(int TimeToWait)
...
-        
+        }
+
         public void WaitAndClickResturantList(int TimeToWait)
         {
             _restaurantsList.WaitAndClick(driver, TimeToWait);
         }
 
-        public void WaitAndClicBeerTag(int TimeToWait)
+        public void WaitAndClickBeerTag(int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs]: **t**imeToWait

File: EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs [hunk 26]
```diff
@@ -34,21 +35,21 @@ public void WaitAndClickJonsonDetails(int TimeToWait)
         public void WaitAndClickJonsonMenu(int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs]: **t**imeToWait

File: EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs [hunk 27]
```diff
@@ -9,23 +9,24 @@ public class RestaurantsPage : BasePage
...
         [FindsBy(How = How.XPath, Using = "//a[@href='/restaurants/2']")]
         private IWebElement _johnsonDetails;
 
         [FindsBy(How = How.XPath, Using = "//a[@href='/restaurant/2/menu/3']/..")]
-        private IWebElement _johnsonMenu;                    
-                          
+        private IWebElement _johnsonMenu;
+
+
         public void WaitAndClickJonsonDetails(int TimeToWait)
```
Comment (dmatsev) [inline EasyRestProjectNetTeam2/EasyRestPages/RestaurantsPage.cs]: **t**imeToWait

**Review by dmatsev — APPROVED** [review]

**Review by AhnessP — APPROVED** [review]

**Pull request merged**